src/text-expander-element.ts

```diff
@@ -90,6 +90,7 @@ class TextExpander {
     const menu = this.menu
     if (!menu || !this.combobox) return
     this.menu = null
+    this.combobox = null
```

Comment:
In line 97 we call `this.combobox.destroy()` I think it may be a problem.

Reply:
I looked at the [combobox-nav package at the `destroy()` method](https://github.com/github/combobox-nav/blob/main/src/index.ts#L32), looks like it removed a bunch of attributes and removes eventlisteners. i still think it should be set to null so i moved that to end of `deactivate()` method.

Reply:
Yes, I think it is safe to set it to `null` as it is reset on `activate` anyway.